lib/constants.js

```diff
@@ -57,8 +57,8 @@ const dayOfWeek = {
 
 const thermostatRunningStates = {
     0: 'idle',
-    1: 'heat',
-    2: 'cool',
+    1: 'heat_no_fan',
+    2: 'cool_no_fan',
```

Comment:
Please don't change these since this will introduce a breaking change.

Reply:
I guess the problem is that means that there's no way of distinguishing the regular heat state with this special heat state.  Is there a way I can implement something specific to the centralite device that interprets this value differently?  The values are different, and the states they represent according to the zigbee spec is different - it's just unfortunate that they were given the same text value in the converters.